src/main/resources/el-GR.yml

```diff
@@ -0,0 +1,105 @@
+el-GR:
+  faker:
+    job:
+      field:
+      - "Μάρκετινγκ"
+      - "Πληροφορική"
+      - "Λογιστικά"
+      - "Διοίκηση"
+      - "Διαφήμιση"
+      - "Τραπεζικά"
+      - "Κοινωνικές Υπηρεσίες"
+      - "Κατασκευές"
+      - "Συμβουλευτικές"
+      - "Σχέδιο"
+      - "Εκπαίδευση"
+      - "Γεωργία"
+      - "Δημόσιες Υπηρεσίες"
+      - "Υγεία"
+      - "Τουρισμός - Εστίαση"
+      - "Νομικά"
+      - "Παραγωγή Προϊόντων"
+      - "Μεταλλεία"
+      - "Ακίνητα"
+      - "Λιανεμπόριο"
+      - "Πωλήσεις"
+      - "Τεχνολογία"
+      seniority:
```

Comment:
Did you try to translate these via Google or Deepl? (Maybe that'd help....or maybe some just don't make sense in Greek and should be trimmed?)

Reply:
Yes, I tried with Google Translate but the output was not what I expected for. I just tried all of them as an input in Deepl as well and it also left a lot of them untranslated.

To be honest, I treated the new el-GR.yml as a 1-1 translation for job.yml. Since the library is just a data "faker" and not a translation tool, you are right that I can trim the ones that do not fit. 
There are still ones which are used mostly in their English variation (e.g. Lead, Senior, Junior, etc), so for these it seems more natural to be kept as is.

Thanks for the review and the tip @kingthorin ! I will come back later with the changes.

Reply:
The file was updated. 
The English terms are used to express `seniority`. I decided to not translate them because they will sound silly even in Greek context. 😄  
Even words that were translated before they were removed as out of scope because they are not used to express `seniority` of a job in the Greek language.